CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
   - `PartialUpdateInstance` can be used to update the instance's name
   - `InstanceParameters` has a new setting `MaxDiskThreshold`
   - `ListMetricsResponse` has new fields regarding ephemeral disk
+- `mongodbflex`: [v0.10.0](services/mongodbflex/CHANGELOG.md#v0100-2024-01-05)
```

Comment:
Is this already released? The date doesn't match

Reply:
Yes, the tag was already created